src/components/genes/GenesSubscriber.js

```diff
@@ -50,30 +50,29 @@ export default function GenesSubscriber(props) {
     setFeatureFilter: setGeneFilter,
     setFeatureHighlight: setGeneHighlight,
     setObsColorEncoding: setCellColorEncoding,
-  }] = useCoordination(COMPONENT_COORDINATION_TYPES.genes, coordinationScopes);
+  }] = useCoordination(COMPONENT_COORDINATION_TYPES[ViewType.FEATURE_LIST], coordinationScopes);
 
-  const [urls, addUrl, resetUrls] = useUrls();
-  const [
-    isReady,
-    setItemIsReady,
-    setItemIsNotReady, // eslint-disable-line no-unused-vars
-    resetReadyItems,
-  ] = useReady(
-    GENES_DATA_TYPES,
-  );
+  const variablesLabel = variablesLabelOverride || featureType;
 
-  // Reset file URLs and loader progress when the dataset has changed.
-  useEffect(() => {
-    resetUrls();
-    resetReadyItems();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [loaders, dataset]);
+  const title = titleOverride || `${capitalize(variablesLabel)} List`;
+
+  const [urls, addUrl] = useUrls(loaders, dataset);
 
   // Get data from loaders using the data hooks.
-  const [attrs] = useExpressionAttrs(
-    loaders, dataset, setItemIsReady, addUrl, true,
+  // TODO: support multiple feature labels using featureLabelsType coordination values.
```

Comment:
Yes it is for geneAlias. in theory there could be multiple types of gene aliases, but i think that can be addressed in a separate PR since it would be a new feature.